### PR TITLE
Remove non-ascii characters from internal code comments

### PR DESCRIPTION
**Pull request by dsherret**
In Deno we now require 7-bit ascii for our internal code. When TypeScript builds, some of these characters end up in the build output. It seems like there weren't that many instances of non-ascii characters in TypeScript's comments so is it ok to remove them? It makes my life a little easier and makes the code a bit more consistent.

I only did a search for the characters that ended up in the build output in the "src" directory and nothing else.

### CODE REVIEW

**Comment by jakebailey** [comment]
This seems okay, but, why is it a requirement for Deno? Can you exempt vendored code? How can we make sure to not regress? What if we add special characters to document something?

**Comment by dsherret** [comment]
> How can we make sure to not regress?

Don't worry about it on your side. Our code just fails to build and it's easy to workaround. Only about 4 of these comments actually ended up in the output so I think it's really rare.

> What if we add special characters to document something?

At that point I'll just add something to transform them.

> Why is it a requirement for Deno? Can you exempt vendored code?

No, it allows us to use v8 external one-byte strings which drastically reduces our snapshot size. Once one of these characters ends up in the output then it goes to utf-16.

**Comment by jakebailey** [comment]
> No, it allows us to use v8 external one-byte strings which drastically reduces our snapshot size. Once one of these characters ends up in the code then it goes to utf-16.

That's super interesting!

File: src/services/codefixes/addMissingAwait.ts [hunk 1]
```diff
@@ -129,7 +129,7 @@ function getDeclarationSiteFix(context: CodeFixContext | CodeFixAllContext, expr
             }
         });
         // No fix-all because it will already be included once with the use site fix,
-        // and for simplicity the fix-all doesn‘t let the user choose between use-site and declaration-site fixes.
+        // and for simplicity the fix-all doesn't let the user choose between use-site and declaration-site fixes.
```
Comment (RyanCavanaugh) [inline src/services/codefixes/addMissingAwait.ts:132]: Can't stop looking at this ‘ psuedoapostrophe

**Pull request merged**